### PR TITLE
Trdt-544-1: OAS migration without using speccy

### PR DESCRIPTION
**Pull request by sarinajsal**

### CODE REVIEW

File: it/uk/gov/hmrc/transactionalrisking/DocumentationIntegrationSpec.scala [hunk 1]
```diff
@@ -63,12 +63,11 @@ class DocumentationIntegrationSpec
       "the version is unknown" in {
         val response =
           wsClient
-            .url(s"$baseUrl/api/conf/4.4.42/application.yaml")
+            .url(s"$baseUrl/api/conf/4.4.42/applicationHelloWorld.yaml")
```
Comment (basotia) [inline it/uk/gov/hmrc/transactionalrisking/DocumentationIntegrationSpec.scala:66]: Does this need to be HelloWorld?

File: .gitignore [hunk 2]
```diff
@@ -37,3 +37,4 @@ target
 test-result
 tmp
 .vscode/
+.node_modules
```
Comment (basotia) [inline .gitignore:40]: are we still using npm?

File: it/uk/gov/hmrc/transactionalrisking/DocumentationIntegrationSpec.scala (hunk 1 above)
Comment (sarinajsal) [reply it/uk/gov/hmrc/transactionalrisking/DocumentationIntegrationSpec.scala:66]: it needs to be application.yaml, I think this was due to some refactoring

File: .gitignore (hunk 2 above)
Comment (sarinajsal) [reply .gitignore:40]: I dont think so but not sure, Im not sure whether I can just get rid of all node modules as they seem to remain ignored in main

Comment (basotia) [reply .gitignore:40]: its fine we can leave it here at the moment

**Review by basotia — APPROVED** [review]

**Pull request merged**